akka-diagnostics/src/main/scala/akka/diagnostics/ConfigChecker.scala

```diff
@@ -795,6 +804,47 @@ class ConfigChecker(system: ExtendedActorSystem, config: Config, reference: Conf
       else Nil
     }
 
+  private def checkRemoteDeployment(): List[ConfigWarning] =
```

Comment:
wrong method name since this is about remote vs cluster

Reply:
changed